text/src/autogluon/text/automm/predictor.py

```diff
@@ -41,6 +41,7 @@
     average_checkpoints,
     infer_metrics,
     get_config,
+    NoParsingFilter,
```

Comment:
Consider to rename it LightningWarningFilter.

Reply:
Better to be LightningInfoFilter since we use it filter info messages.